modules/core/include/opencv2/core/fast_math.hpp

```diff
@@ -130,18 +136,8 @@ cvRound( double value )
  */
 CV_INLINE int cvFloor( double value )
 {
-#if (defined _MSC_VER && defined _M_X64 || (defined __GNUC__ && defined __SSE2__ && !defined __APPLE__)) && !defined(__CUDACC__)
```

Comment:
why is this part changed? cvFloor() and cvCeil() (together with cvRound) are some of the most performance-critical functions in the library; we want to keep them maximally efficient. Can you please revert the changes?